src/main/groovy/org/shipkit/internal/notes/header/HeaderRemover.java

```diff
@@ -0,0 +1,46 @@
+package org.shipkit.internal.notes.header;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HeaderRemover {
```

Comment:
I'm wondering if we can make it simpler.

How about we merge the functionality of FileUtil.appendToTop, HeaderRemover and HeaderProvider. A single type that handles appending to the file with a header information. "FileUtil.appendToTop" already loads the entire file content, so you can probably simplify the implementation and just operate on Strings instead of creating buffered reader/writer classes. Also, why do we need to open file, read it, remove header, write it back, then we do the same, open the file again, read it, insert release note to the top. In general, I don't mind decoupled functionalities (e.g. removal of header is decoupled from writing the content). However, in this instance I think that an alternative implementation would give us much simpler design.